src/content/docs/en/reference/experimental-flags/svg.mdx

```diff
@@ -0,0 +1,187 @@
+  }
+})
+```
+
+## Configuration
+
+### `optimize`
+
+**Type:** `boolean`<br />
+**Default:** `true`
+
+Whether to enable SVG optimization using SVGO during build time.
+
+When enabled, all imported SVG files will be optimized for smaller file sizes and better performance while maintaining visual quality.
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true
+    }
+  }
+})
+```
+
+### `svgoConfig`
+
+**Type:** `Config` (SVGO configuration object)<br />
+**Default:** `{}`
+
+Configuration object passed directly to SVGO for customizing SVG optimization.
+
+See [SVGO documentation](https://svgo.dev/docs/preset-default/) for available options and plugins.
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true,
+      svgoConfig: {
+        plugins: [
+          'preset-default',
+          {
+            name: 'removeViewBox',
+            active: false
+          }
+        ]
+      }
+    }
+  }
+})
+```
+
+## Usage
+
+Once enabled, SVG optimization will automatically apply to all SVG files imported in your project:
+
+```astro title="src/pages/index.astro"
+---
+import Logo from '../assets/logo.svg';
+---
+
+<Logo />
+```
+
+The SVG will be optimized during the build process, resulting in smaller file sizes in your production build.
+
+## Common use cases
+
+### Preserve specific attributes
+
+You may want to preserve certain SVG attributes that SVGO removes by default:
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true,
+      svgoConfig: {
+        plugins: [
+          'preset-default',
+          {
+            name: 'removeViewBox',
+            active: false // Preserve viewBox attribute
+          }
+        ]
+      }
+    }
+  }
+})
+```
+
+### Remove specific elements
+
+Remove unwanted elements like metadata or hidden layers:
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true,
+      svgoConfig: {
+        plugins: [
+          'preset-default',
+          {
+            name: 'removeHiddenElems',
+            active: true
+          }
+        ]
+      }
+    }
+  }
+})
+```
+
+### Custom precision
+
+Control the precision of numeric values in path data:
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true,
+      svgoConfig: {
+        floatPrecision: 2
+      }
+    }
+  }
+})
+```
+
+## Error handling
+
+If SVGO optimization fails for any reason, Astro will gracefully fall back to using the original, unoptimized SVG content. A warning will be logged to the console, but your build will continue without errors.
+
+## Performance considerations
+
+SVG optimization happens during the build process, not at runtime. This means:
+
+- **Development:** SVGs are not optimized during development for faster rebuild times
```

Comment:
This doesn't feel like a cohesive list of three related things: dev, prod, then... something else? And not sure Performance Considerations will be the right heading, since that implies "Things you have to think about to manage performance" but really, this is just "how it works." Will think about this one!